src/Results/Components/Results.tsx

```diff
@@ -84,7 +79,7 @@ function Component(props: Props) {
       </>
     );
   } else {
-    display = (
+    displayResult = (
```

Comment:
I would suggest to think whether this and the other result could be a component instead. It might be easier to read the logic on place  displayResult is used rather than how it is defined.